sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -259,6 +239,12 @@ RetT *getOrBuild(KernelProgramCache &KPCache, KeyT &&CacheKey,
 
 static bool isDeviceBinaryTypeSupported(const context &C,
                                         RT::PiDeviceBinaryType Format) {
+  backend CBackend = (detail::getSyclObjImpl(C)->getPlugin()).getBackend();
```

Comment:
```suggestion
  const backend CBackend = (detail::getSyclObjImpl(C)->getPlugin()).getBackend();
```

Reply:
Addresed in ec971f97ec8cdcdcd9d7404dab73a880bc3ac4d8